doc/src/overview.xml

```diff
@@ -675,10 +675,10 @@ b2 toolset=gcc variant=debug optimization=space
             <listitem>
               <para>Changes the build directories for all project roots being built. When
               this option is specified, all Jamroot files must declare a project name.
-              The build directory for the project root will be computed by concatanating
+              The build directory for the project root will be computed by concatenating
               the value of the <option>--build-dir</option> option, the project name
               specified in Jamroot, and the build dir specified in Jamroot
-              (or <literal>bin</literal>, if none is specified).
+              (or <literal>bin.v2</literal>, if none is specified).
```

Comment:
The original version is correct.  bin.v2 is specified by the superproject's Jamroot.